ext/BijectorsEnzymeExt.jl

```diff
@@ -0,0 +1,14 @@
+module BijectorsEnzymeExt
+
+if isdefined(Base, :get_extension)
+    using Enzyme: @import_frule, @import_rrule
+    using Bijectors: find_alpha
+else
+    using ..Tapir: @import_frule, @import_rrule
+    using ..Bijectors: find_alpha
+end
+
+@import_rrule typeof(find_alpha) Float64 Float64 Float64
+@import_frule typeof(find_alpha) Float64 Float64 Float64
```

Comment:
Same comment as in https://github.com/TuringLang/Bijectors.jl/pull/319#discussion_r1657123613: I think this is too restrictive.

Reply:
Yeah just make this real or whatever the original fn is defined on.

No restrictions on support from enzyme side

Reply:
Done